repository/Seaside-Core.package/WAResponse.class/instance/writeStatusOn..st

```diff
@@ -1,11 +1,12 @@
 writing
-writeStatusOn: aStream 
-	aStream nextPutAll: self httpVersion.
-	aStream
-		nextPut: $ ;
-		nextPutAll: self status greaseString.
-	self message isNil ifFalse:  [
-		aStream
-			nextPut: $ ;
-			nextPutAll: self message ].
-	aStream crlf
\ No newline at end of file
+writeStatusOn: aStream
+  aStream nextPutAll: self httpVersion.
+  aStream
+    nextPutAll: $  greaseString;
+    nextPutAll: self status greaseString.
+  self message isNil
+    ifFalse: [ 
+      aStream
+        nextPutAll: $  greaseString;
```

Comment:
Is this really necessary? Not that it's a huge deal, but the iteration overhead for single character + an additional method activation seem a bit ugly.